src/game/title_screen.c

```diff
@@ -1729,18 +1729,18 @@ static void Task_LensFlareAnim(void)
     if (!(lensFlare->animFrame & 1)) {
         for (i = 0; i < 8; i++) {
             sprite = &lensFlare->sprites[i];
-            config180 = &lensFlare->unk180[i];
+            transforms = &lensFlare->transforms[i];
```

Comment:
Minor, but I think this var should just be `transform`?

Reply:
On it. It was a copy-paste, will keep it for the array though, as it makes sense, I think.

Reply:
Yep!